JOSESwift/Sources/JWEHeader.swift

```diff
@@ -89,7 +89,7 @@ public struct JWEHeader: JOSEHeader {
 // Header parameters that are specific to a JWE Header.
 public extension JWEHeader {
     /// The algorithm used to encrypt or determine the value of the Content Encryption Key.
-    public var algorithm: AsymmetricKeyAlgorithm? {
+    var algorithm: AsymmetricKeyAlgorithm? {
```

Comment:
Why the visibility changes in this file?

Reply:
Symbols in a public extension are public by default.